example/src/main/java/com/makeramen/roundedimageview/example/PicassoFragment.java

```diff
@@ -77,6 +77,7 @@ public PicassoAdapter(Context context) {
           .borderColor(Color.BLACK)
           .borderWidthDp(3)
           .oval(false)
+          .picasso()
```

Comment:
While I like this pattern it's a breaking API change that would be a major version bump, which is fine. But I would like to bundle any other API changes together, so probably would want to wait until we have a new API for `Corners` as well before I'd release a cut of 3.x.